lib/utils/common-utils.ts

```diff
@@ -0,0 +1,7 @@
+export const KEY = {
+    ARROW_LEFT: "ArrowLeft",
+    ARROW_RIGHT: "ArrowRight",
+    ENTER: "Enter",
+    SPACE: " ",
+    TAB: "Tab",
+};
```

Comment:
nit-pick: please add a line break in the end, so that git doesn't complain with  ⊖ sign :)